packages/components/src/lib/queries/lexical-query.js

```diff
@@ -99,30 +109,24 @@ export default class LexicalQuery extends Query {
   }
 
   async getData () {
-    this.languageID = this.selector.languageID
-    if (this._hasLexemes) {
-      // Some lexical data is available, do a partial data retrieval
-      await this.partialDataRetrieval()
-    } else {
```

Comment:
This code was needed for the new lexical query workflow. All related functionality has been moved to `Lexis` now, and this path is no longer needed in the `LexicalQuery`.